ansible/roles/nim-waku/templates/docker-compose.yml.j2

```diff
@@ -45,6 +45,7 @@ services:
       --persist-messages=true
       --lightpush=true
       --swap={{ nim_waku_swap_protocol_enabled | lower }}
+      --max-connections=300
```

Comment:
Could we parametrize this maybe? Also, are we sure the hosts can handle that much?

I don't mind increasing it, just that we could possibly see performance issues if too many connections are established, not sure tho.

Reply:
Right. Parametrized in 9b565f4.

> Also, are we sure the hosts can handle that much?

Valid point. They _should_ be able to handle it based on a similar setup in nimbus-eth2, but I've changed the default, prod and test max connections to `150` for now, which is [the maximum I have benchmarked](https://vac.dev/waku-v1-v2-bandwidth-comparison). This should be more than enough for the time being. We haven't really seen more than 25 connections in the last ~2 months.